modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -380,6 +380,11 @@ const register = (editor: Editor): void => {
     default: false
   });
 
+  registerOption('format_wrap_noneditable_selectors', {
```

Comment:
I have added a new option that can be used by integrators. Would we be happy with `format_wrap_noneditable_selectors` as the option name?